### PR TITLE
Grey out merge commits

### PR DESCRIPTION
**Pull request by soudy**
Implement #2696
Greyed out commit

### CODE REVIEW

File: templates/repo/commits_table.tmpl [hunk 1]
```diff
@@ -35,7 +35,12 @@
 								<img class="ui avatar image" src="{{AvatarLink .Author.Email}}" alt=""/>&nbsp;&nbsp;{{.Author.Name}}
 							{{end}}
 						</td>
+
+						{{if gt .ParentCount 1}}
+						<td class="message collapsing has-emoji grey text">
```
Comment (Unknwon) [inline templates/repo/commits_table.tmpl]: Why don't you just put `grey text` inside the `if` block, instead of the whole line?

Comment (soudy) [inline templates/repo/commits_table.tmpl]: @Unknwon because stretching out HTML tags with if statements makes it way less readable imo

Comment (Unknwon) [inline templates/repo/commits_table.tmpl]: Disagree. Duplicated majority of content is worse, unless case branches are a lot. In this case we only have two branches.

**Comment by soudy** [comment]
@Unknwon I've updated it to your likings

**Comment by Unknwon** [comment]
Thanks!

**Pull request merged**